frontend/views/components/modal/ModalTemplate.vue

```diff
@@ -14,10 +14,10 @@
       .c-modal-content(ref='card' v-if='modalIsActive')
         header.c-modal-header(
           :class='{ "has-subtitle": $scopedSlots.subtitle }'
-          v-if='$scopedSlots.title || $scopedSlots.subTitle'
+          v-if='$scopedSlots.title || $scopedSlots.is-subtitle'
```

Comment:
I think this was a wrong find-replace. it should be subTitle.

Reply:
Fixed. Thanks